lib/matplotlib/backends/backend_gtk.py

```diff
@@ -1003,13 +999,13 @@ def on_combobox_lineprops_changed(self, item):
         if marker is None: marker = 'None'
         self.cbox_markers.set_active(self.markerd[marker])
 
-        r,g,b = colorConverter.to_rgb(line.get_color())
-        color = gtk.gdk.Color(*[int(val*65535) for val in (r,g,b)])
+        rgba = mcolors.to_rgba(line.get_color())
```

Comment:
This is a section of the code that I was waiting for MEP27 to get rid of it. I plan to introduce the same functionality as generic tool for all the backends.

This class `DialogLineprops` is used only by one example: `examples/user_interfaces/lineprops_dialog_gtk.py`

The class was copy/paste to Gtk3 but last time I checked it wasn't working.

My advice would be, if the example runs, then it is fine. (I don't have a working gtk2 environment anymore to test)